jsonrpc/dispatcher.go

```diff
@@ -434,6 +435,16 @@ func (d *Dispatcher) getBlockHeaderImpl(number BlockNumber) (*types.Header, erro
 	}
 }
 
+func (d *Dispatcher) isAccountAvailable(addr types.Address) (isAvailable bool) {
```

Comment:
Minor nitpick, there is no need to use a named return if we're not utilizing the actual name of the return variable (should return `bool`, not `isAvailable bool`, since we don't leverage `isAvailable` within the method)

Reply:
This method is no more required, Removed It. Commit-[link](https://github.com/0xPolygon/polygon-sdk/pull/148/commits/41c54a5ed786a9241552993650ae88bdfcfdd47b)